### DISCUSSION TITLE
请问这种报错因为没有配置什么造成的，可以帮忙看下吗

### DISCUSSION THREAD

**Discussion by old-zsh** [umijs/umi#10113] [Q&A]
![2](https://user-images.githubusercontent.com/20679549/208564613-801b228c-844f-4193-b032-b61cf9191ee4.png)

**Comment by fz6m** [comment]
语法有问题，给最小复现看看，是哪个语法有问题，二分定位下。

**Reply by old-zsh** [reply]
> 语法有问题，给最小复现看看，是哪个语法有问题，二分定位下。

我似乎在找到答案了，请问在umi里面如何配置 类似webpack下面的配置
configureWebpack: {

    module: {

      rules: [

          {

              test: /\.mjs$/,

              include: /node_modules/,

              type: "javascript/auto"

          },

      ]

  }

  }
